agent/tool-scripts/postprocess/disk-postprocess

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+# Check for an alternate tools library path for testing
+my $_test_alt_tools_lib;
+BEGIN {
+        my $_pbench_tspp_dir = $ENV{'pbench_tspp_dir'};
+        $_test_alt_tools_lib=$ENV{_TEST_ALTERNATE_TOOLS_LIBRARY};
+        if (not defined $_test_alt_tools_lib or not -d $_test_alt_tools_lib) {
+                $_test_alt_tools_lib = "$_pbench_tspp_dir";
+        }
+}
+use lib "$_test_alt_tools_lib";
+no lib ".";
+use GenData qw(gen_data);
+
+my $dir=$ARGV[0];
+my $mounted;
+my %disk_data;
+my $timestamp;
+my $timestamp_ms;
+
+# read the iostat.txt
+open(DISK_TXT, "$dir/disk-stdout.txt") || die "could not find $dir/disk-stdout.txt\n";
+while (my $line = <DISK_TXT>) {
+	chomp $line;
+	# Each sample should start with a timestamp.
+	# Javascript requires the timestamp to be in milliseconds.
+
+	#1493489459345 Filesystem              Mounted on     Type      Inodes IUsed   IFree 1K-blocks    Used    Avail File
+	#1493489459345 devtmpfs                /dev           devtmpfs 1019666   392 1019274   4078664       0  4078664 -
+	#1493489459345 tmpfs                   /dev/shm       tmpfs    1022017     1 1022016   4088068       0  4088068 -
+
+	if ( $line =~ /(\d+)\s+(\S+)\s+(\S+)\s+(\S+)\s+(\d+)+(\d+)\s+(\d+)\s+(\d+)\s+(\d+)\s+(\d+)\s+(\S+)/ ) {
```

Comment:
The regex here seems fragile, as it appears to skip the header line because the `target` column header uses `Mounted on`.  It would have been nice to remove that during collection, or to explicitly ignore a line that begins with `<ts> Filesystem`.